### ISSUE TITLE
Hide playhead until waveform is ready

### ISSUE THREAD

**Issue by thijstriemstra** [katspaugh/wavesurfer.js#306]
The playhead is always rendered, no matter if a `load` call has been issued or not (see screenshot below). I think it should be hidden until a waveform is ready.

![schermafbeelding 2014-12-03 om 19 04 30](https://cloud.githubusercontent.com/assets/305679/5285883/67cd13da-7b17-11e4-9273-0358e765fa17.png)

**Comment by thijstriemstra** [comment]
Simple workaround is hiding the element that contains the playhead:
```javascript
wavesurferElement.getElementsByTagName("wave")[1].style.display = 'none';
```

**Comment by katspaugh** [comment]
I tend to agree that playhead shouldn't be visible without the audio. Can you make a pull request?

**Comment by thijstriemstra** [comment]
@katspaugh sure, please assign this ticket to me.

**Comment by thijstriemstra** [comment]
Released in 1.0.25.
